solution/index.js

```diff
@@ -0,0 +1,758 @@
+                                                                    && taskObj.date === liTask.childNodes[1].childNodes[1].textContent
+                                                                    && taskObj.time === liTask.childNodes[1].childNodes[2].textContent
+                                                                    ?  data["inProgress"].splice(i,1) 
+                                                                    :  console.log(taskObj));
+    }
+    
+
+    //remove done tasks from data collectors
+    if(liTask.parentElement.id === "done-tasks"){
+        //delete done task from data object
+        data["done"].forEach( (taskObj, i) => taskObj.task === liTask.firstChild.textContent
+                                                                    && taskObj.date === liTask.childNodes[1].childNodes[1].textContent 
+                                                                    && taskObj.time === liTask.childNodes[1].childNodes[2].textContent 
+                                                                    ?  data["done"].splice(i,1)
+                                                                    :  console.log(taskObj) );
+    }
+
+     //remove from localstorage todo === add updated data boject
+     setDataToLocalStorage();
+    
+
+ }
+    
+
+    
+
+    
+
+
+
+    // listItems.forEach((list) => { //observing the changes for every list
+    // observer.observe( list, {childList: true }); // observing for list items which being added
+    // });
+/** ~~ !!! STOP THE MUTANT WHEN USER NOT MAKING AN EVENT RELATED TO IT!!! ~~ */
+
+document.addEventListener( 'keydown', (e) => { // detectin keyboard events are only on input type elements so i detect it through window object
+    
+    if( e.altKey || e.altGraphKey )
+    {   
+        e.preventDefault(); //preventing default so it wont switch to other objects and wont capture the windows events!
+
+        if(isHover && e.key === '1' && currentHoveredEl.parentElement.parentElement.parentElement.id !== "todo-tasks-section")    
+        {
+            localStorageTaskRemover(currentHoveredEl);//remove fro local storage
+            data["todo"].unshift({task: currentHoveredEl.firstChild.textContent, date:currentHoveredEl.childNodes[1].childNodes[1].textContent  , time:currentHoveredEl.childNodes[1].childNodes[2].textContent}) //update data in the new current thing of list
+            setDataToLocalStorage();//update local storage
+            if(todoList.firstElementChild === undefined) //if it doesnt have a  child, append it.
+            {
+                eleDOMAppender(todoList,currentHoveredEl.parentElement);  //if current task is hovered and not in the same list of the destination list then pass it over.. 
+            }
+            else{ //if it does  have a child insert the generated li from user to the top of the ul
+                todoList.insertBefore(currentHoveredEl, todoList.firstChild);
```

Comment:
Why is this entire code block duplicated 3 times for every list? Write a function!